### ISSUE TITLE
Dynamic global DNS/domains settings via dbus/resolvectl

### ISSUE THREAD

**Issue by es-fabricemarie** [systemd/systemd#21948]
It does not appear to be possible to set global level configuration such as global DNS resolver or domain via dbus interface or via `resolvectl`. Both dbus interface and `resolvectl` take an *iface_index* parameter, but they both refuse `0` (global) as the index.

I would like to be able to change global settings dynamically, without having to restart systemd-resolved.

Apparently, currently the only way is to alter `/etc/systemd/resolved.conf` and restart `systemd-resolved.service`.

**Comment by Ypnose** [comment]
I agree, that feature would be extremely useful. Will it be supported ?
